Gruntfile.js

```diff
@@ -82,10 +82,9 @@ function configureGrunt(grunt) {
     grunt.registerTask("jasmine-node:test-on-server", function() {
         var done = this.async();
 
-        function afterJasmineNodeCompletes(exitCode) {
-            if (exitCode !== 0) {
-                grunt.fail();
-                return;
+        function preserveExitCode(exitCode) {
```

Comment:
just curious - what do you mean by preserveExitCode? does grunt.fail.fatal do that?